rke/k8s_rke_system_images.go

```diff
@@ -8375,9 +8375,9 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.21.1",
 		},
 		// Enabled in Rancher v2.7.1
```

Comment:
Let's change the comment here to reference v2.7.2, please.

Reply:
done